CP01CNSM/CP01CNSM_R00004_ingest.csv

```diff
@@ -1,50 +1,50 @@
+#mi.dataset.driver.ctdbp_cdef.ctdbp_cdef_recovered_driver,/omc_data/whoi/OMC/CP01CNSM/R00004/instruments/dcl37/CTDBP*/SBE*.hex,CP01CNSM-MFD37-03-CTDBPD000,recovered_inst,Not Deployed,
+#mi.dataset.driver.dosta_abcdjm.dcl.dosta_abcdjm_dcl_recovered_driver,/omc_data/whoi/OMC/CP01CNSM/R00004/cg_data/dcl37/dosta2/*dosta*.log,CP01CNSM-MFD37-04-DOSTAD000,recovered_host,Not Deployed,
+#mi.dataset.driver.zplsc_c.dcl.zplsc_c_dcl_telemetered_driver,/omc_data/whoi/OMC/CP01CNSM/R00004/cg_data/dcl37/zplsc/*.zplsc.log,CP01CNSM-MFD37-07-ZPLSCC000,recovered_host,Not Deployed,
 #mi.dataset.driver.cg_cpm_eng.cpm.cg_cpm_eng_cpm_recovered_driver,/omc_data/whoi/OMC/CP01CNSM/R00004/cg_data/cpm2/syslog/cpm_status.txt,CP01CNSM-RIC21-00-CPMENG000,recovered_host,Not Deployed,
 mi.dataset.driver.cg_dcl_eng.dcl.cg_dcl_eng_dcl_recovered_driver,/omc_data/whoi/OMC/CP01CNSM/R00004/cg_data/dcl26/syslog/*.syslog.log,CP01CNSM-RID26-00-DCLENG000,recovered_host,Available,
-mi.dataset.driver.velpt_ab.dcl.velpt_ab_dcl_recovered_driver,/omc_data/whoi/OMC/CP01CNSM/R00004/cg_data/dcl26/VELPT_snAQD-8532/*.velpt1.log,CP01CNSM-RID26-04-VELPTA000,recovered_host,Available,
-mi.dataset.driver.velpt_ab.velpt_ab_recovered_driver,/omc_data/whoi/OMC/CP01CNSM/R00004/cg_data/dcl26/VELPT*/*.aqd,CP01CNSM-RID26-04-VELPTA000,recovered_inst,Available,
-#mi.dataset.driver.phsen_abcdef.dcl.phsen_abcdef_dcl_recovered_driver,/omc_data/whoi/OMC/CP01CNSM/R00004/cg_data/dcl26/PHSEN_snP0068/*.phsen1.log,CP01CNSM-RID26-06-PHSEND000,recovered_host,Not Deployed,
-#mi.dataset.driver.phsen_abcdef.phsen_abcdef_recovered_driver,/omc_data/whoi/OMC/CP01CNSM/R00004/cg_data/dcl26/PHSEN_*/SAMI*.txt,CP01CNSM-RID26-06-PHSEND000,recovered_inst,Not Deployed,
-mi.dataset.driver.nutnr_b.dcl_full.nutnr_b_dcl_full_recovered_driver,/omc_data/whoi/OMC/CP01CNSM/R00004/cg_data/dcl26/NUTNR_sn284/*.nutnr.log,CP01CNSM-RID26-07-NUTNRB000,recovered_host,Available,
-mi.dataset.driver.nutnr_b.nutnr_b_recovered_driver,/omc_data/whoi/OMC/CP01CNSM/R00004/cg_data/dcl26/NUTNR_sn284/SCH*.DAT,CP01CNSM-RID26-07-NUTNRB000,recovered_inst,Available,
-mi.dataset.driver.spkir_abj.dcl.spkir_abj_dcl_recovered_driver,/omc_data/whoi/OMC/CP01CNSM/R00004/cg_data/dcl26/SPKIR_sn291/*.spkir.log,CP01CNSM-RID26-08-SPKIRB000,recovered_host,Available,
+mi.dataset.driver.velpt_ab.dcl.velpt_ab_dcl_recovered_driver,/omc_data/whoi/OMC/CP01CNSM/R00004/cg_data/dcl26/velpt1/*.velpt1.log,CP01CNSM-RID26-04-VELPTA000,recovered_host,Available,
+mi.dataset.driver.velpt_ab.velpt_ab_recovered_driver,/omc_data/whoi/OMC/CP01CNSM/R00004/instruments/dcl26/VELPT*/*.aqd,CP01CNSM-RID26-04-VELPTA000,recovered_inst,Available,
+#mi.dataset.driver.phsen_abcdef.dcl.phsen_abcdef_dcl_recovered_driver,/omc_data/whoi/OMC/CP01CNSM/R00004/cg_data/dcl26/phsen1//*.phsen1.log,CP01CNSM-RID26-06-PHSEND000,recovered_host,Not Deployed,
```

Comment:
It wasn't deployed so it doesn't really matter, but there is an extra "/" after phsen1